tests/test_job.py

```diff
@@ -1291,6 +1291,18 @@ def test_job_wait_for_close(hq_env: HqEnv):
     assert r == 0
 
 
+def test_job_wait_for_open_job_without_close(hq_env: HqEnv):
+    hq_env.start_server()
+    hq_env.command(["job", "open"])
+
+    hq_env.command(["job", "wait", "1", "--without-close"])
+
+    hq_env.start_worker()
+    hq_env.command(["submit", "--job=1", "--", "sleep", "0"])
```

Comment:
Let's use at least e.g. `sleep 5` here, and wait for the task to be running, to make sure that we test the situation where some tasks are running when we invoke `wait --without-close`.

Reply:
Then tests will last for 5 seconds, but ok, I will try add some more sophisticated testing that wait did not terminate immediately.

Reply:
I know, but it's not such a big problem, as we run tests in parallel anyway (e.g. `python -m pytest -n64`).